File: VisitBosnia/VisitBosnia.Model/SearchObjects/AppUserFavouriteSearchObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VisitBosnia.Model.Requests
{
    public class AppUserFavouriteSearchObject 
    {
        public bool IncludeAppUser { get; set; }
        public bool IncludeTouristFacility { get; set; }
        public int? AppUserId { get; set; }
        public int? TouristFacilityId { get; set; }
    }
}

File: VisitBosnia/visit_bosnia_mobile/lib/model/appUserFavourite/app_user_favourite_search_object.dart
class AppUserFavouriteSearchObject {
  bool? includeAppUser;
  bool? includeTouristFacility;
  int? appUserId;
  int? touristFacilityId;

  AppUserFavouriteSearchObject(
      {this.includeAppUser,
      this.includeTouristFacility,
      this.appUserId,
      this.touristFacilityId});

  AppUserFavouriteSearchObject.fromJson(Map<String, dynamic> json) {
    includeAppUser = json['includeAppUser'];
    includeTouristFacility = json['includeTouristFacility'];
    appUserId = json['appUserId'];
    touristFacilityId = json['touristFacilityId'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['includeAppUser'] = this.includeAppUser;
    data['includeTouristFacility'] = this.includeTouristFacility;
    data['appUserId'] = this.appUserId;
    data['touristFacilityId'] = this.touristFacilityId;
    return data;
  }
}
